pkg/ls/fileinfo_unix.go

```diff
@@ -42,9 +44,20 @@ func FromOSFileInfo(path string, fi os.FileInfo) FileInfo {
 	// A filesystem with a bug will result
 	// in sys not being the right type.
 	// This turns out to be surprisingly messy to test.
-	UID, GID, rdev := uint32(math.MaxUint32), uint32(math.MaxUint32), uint64(math.MaxUint64)
+	uid, gid, rdev := uint32(math.MaxUint32), uint32(math.MaxUint32), uint64(math.MaxUint64)
+	var aTime, cTime, birthTime time.Time
+	var dev, ino, nLink uint64
+	var blkSize, blocks int64
 	if s, ok := fi.Sys().(*syscall.Stat_t); ok {
-		UID, GID, rdev = s.Uid, s.Gid, uint64(s.Rdev)
```

Comment:
This changed I don't quite get, UID is User ID, and I thought the convention as upper case. The Uid is from long ago when this rule was not quite agreed-upon.

Reply:
UID and GID are local variables and not exported so they should begin with lowercase, so this probably should be uID and gID.

Reply:
I understand the capitalization rule for exported structure members, but I have never seen such a rule for function variables, is that documented somewhere?

Reply:
I believe if name is not exported it should be in lowercase. But I don't find explicit wording about that. But from this table https://google.github.io/styleguide/go/decisions.html#initialisms it should uID because of unexported scope.
Exported: IOS, unexported: iOS.

Also it maybe confusing if name is uppercase but not exported, you may think of it as a global variable.

Reply:
but the variable is in function scope. It's not a structure member. So the question of whether it is exported is moot -- it is not in a structure, or at file scope, it is in a function. 

I still don't see that this change is necessary. I'll accept if if you feel it is , however.